test/spec.js

```diff
@@ -0,0 +1,102 @@
+const assert = require('assert');
```

Comment:
tests should be organized in a directory structure similar to the code. Consider how ConseilJS does it: [code](https://github.com/Cryptonomic/ConseilJS/tree/master/src) vs [tests](https://github.com/Cryptonomic/ConseilJS/tree/master/test).